android/src/app/organicmaps/settings/SettingsPrefsFragment.java

```diff
@@ -95,8 +93,6 @@ public boolean onPreferenceChange(Preference preference, Object newValue)
       root.setSummary(R.string.on);
       if (mPrefEnabled != null)
         mPrefEnabled.setTitle(R.string.on);
-      if (mLangInfoLink != null)
-        removePreference(getString(R.string.pref_navigation), mLangInfoLink);
```

Comment:
Behaviour of `mLangInfoLink` is little bit strange. It's not in "Navigation" preferences, so this remove does nothing.